maptivist/src/main/webapp/script.js

```diff
@@ -51,3 +51,70 @@ window.onclick = function(event) {
     form.style.display = "none";
   }
 }
+
+//GOOGLE OAUTH
+
+var GoogleAuth;
+  var SCOPE = 'https://www.googleapis.com/auth/drive.metadata.readonly';
+  function handleClientLoad() {
+    gapi.load('client:auth2', initClient);
+  }
+
+  function initClient() {
+    var discoveryUrl = 'https://www.googleapis.com/discovery/v1/apis/drive/v3/rest';
```

Comment:
And the discovery url for the scope above is:

https://people.googleapis.com/$discovery/rest?version=v1

I found it here:
https://developers.google.com/people/api/rest/